build.sbt

```diff
@@ -1,14 +1,41 @@
 import _root_.play.sbt.PlayScala
+import play.core.PlayVersion
 import sbt.Keys.{testOptions, _}
 import sbt.Tests.{Group, SubProcess}
 import sbt.{Tests, _}
 import uk.gov.hmrc.DefaultBuildSettings._
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
 
+import scala.util.Properties
+
 lazy val appName = "api-example-microservice"
+lazy val appDependencies: Seq[ModuleID] = compile ++ test
+
+lazy val scope: String = "test, it"
+lazy val ComponentTest = config("component") extend IntegrationTest
+def componentTestFilter(name: String): Boolean = name startsWith "component"
```

Comment:
This probably is not needed given there are separate folders.  This used to really only be useful when everything was in the test folder.

Reply:
Hmm... the component tests are not even running right now. I need to fix this.

Reply:
Nope they are all good - just my machine being :hankey:

Reply:
Well that was fun to solve.